Tests/Functional/Controller/ImportTest.php

```diff
@@ -217,22 +217,30 @@ public function test_unassign_remote_to_local_category_without_category_id()
     public function test_unassign_remote_articles_from_local_category()
     {
         /** @var TestClient $client */
-        $client = $this->createBackendClient();
+        $client = self::createBackendClient();
 
         $client->request('POST', 'backend/Import/unassignRemoteArticlesFromLocalCategory');
         $responseData = $this->handleJsonResponse($client->getResponse());
 
         $this->assertTrue($responseData['success']);
+
+        $categoryAssignments = self::getDbalConnection()->fetchAll('SELECT * FROM s_plugin_connect_categories_to_local_categories');
+        $this->assertEmpty($categoryAssignments);
     }
 
     public function test_recreate_remote_categories()
     {
         /** @var TestClient $client */
-        $client = $this->createBackendClient();
+        $client = self::createBackendClient();
 
         $client->request('POST', 'backend/Import/recreateRemoteCategories');
```

Comment:
same as above
this does not test what the function is actually doing
please add some fixtures and assertion based on them
here you need as fixture an entry in connect items with some categories but no connect categories
after the call the connect categories should be there